test/regression.py

```diff
@@ -749,43 +1018,59 @@ def testGaussian_Gaussian09_issue_460_log(logfile):
     assert logfile.data.scfvalues[0][0, 0] == 3.37e-03
     assert numpy.isnan(logfile.data.scfvalues[0][0, 2])
 
+    assert logfile.data.metadata["package_version"] == "2009+D.01"

```

Comment:
Do you think it's a good idea to change the version formatting programs use natively? IIRC this is originally something like '09 Rev D01'.

Reply:
I don't think it's too confusing, and most native formatting is not PEP 440 compliant and will parse to `LegacyVersion`, so it won't sort properly, major/minor versions can't be parsed out of it, etc., which is important if we want to use the version numbers in the parsers, and QCSchema will require it.

Reply:
OK, sounds good. Perhaps it would be prudent to save the legacy version as well as the PEP 440 compliant version we generate from it?

Reply:
Yes, will do. How does `package_legacy_version` sound?

Reply:
Might be better to leave the original name in there, so maybe `legacy_package_version` or `package_version_legacy`?

Reply:
`legacy_package_version` it is.